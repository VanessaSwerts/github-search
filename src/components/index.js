import CustomHeader from './Header/Header'
import CustomFooter from './Footer/Footer'
import Error from './Error/Error'
import Loading from './Loading/Loading'

export {
  CustomHeader,
  CustomFooter,
  Error,
  Loading
}