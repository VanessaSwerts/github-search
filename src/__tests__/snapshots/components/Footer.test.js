import React from 'react'
import { render } from '@testing-library/react'
import { CustomFooter } from '../../../components'

describe('Testes de Snapshot da Footer', () => {

  test('Renderização do componente Footer', () => {
    const { asFragment } = render(<CustomFooter />)

    expect(asFragment(<CustomFooter />)).toMatchSnapshot()
  })

}) 