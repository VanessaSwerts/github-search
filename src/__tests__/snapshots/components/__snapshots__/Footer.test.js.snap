// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testes de Snapshot da Footer Renderização do componente Footer 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    <span>
      Created by VSwerts
    </span>
    <div
      class="contact"
    >
      <a
        href="https://github.com/VanessaSwerts"
      >
        <i
          class="fab fa-github-square"
        />
      </a>
      <a
        href="https://www.linkedin.com/in/vanessaswerts/"
      >
        <i
          class="fab fa-linkedin"
        />
      </a>
    </div>
  </div>
</DocumentFragment>
`;
